app/js/services/services.js

```diff
@@ -710,29 +723,9 @@ KMCServices.directive('loadingWidget', ['requestNotificationChannel', function (
 KMCServices.factory('editableProperties', ['$q', 'api', '$http', function ($q, api, $http) {
 	var deferred = $q.defer();
 	api.then(function () {
-		//for debbuging
-//       return $http.get('js/services/editableProperties.json').then(function(result){
-//           deferred.resolve(result.data);
-//        });
-//
-
-		var method = 'get';
-		var url = window.kWidget.getPath() + 'services.php?service=studioService';
-		if (window.IE < 10) {
-			url += '&callback=JSON_CALLBACK';
-			method = 'jsonp';
-		}
-		$http[method](url).then(function (result) {
-			var data = result.data;
-			if (typeof data == 'object') // json is OK
-				deferred.resolve(result.data);
-			else {
-				cl('JSON parse error of playerFeatures');
-				deferred.reject(false);
-			}
-		}, function (reason) {
-			deferred.reject(reason);
-		});
+      return $http.get('js/services/v3Properties.json').then(function(result){
```

Comment:
@yairans - is this going to stay hard-coded? If so, no need to wait for the API promise...